.aws/src/event-rules/prospect-events/prospectEventRules.ts

```diff
@@ -78,6 +81,13 @@ export class ProspectEvents extends Resource {
 
     // TODO: create a policy function for dev event bridge
     // this.createPolicyForEventBridgeToDevEventBridge();
+
+    createDeadLetterQueueAlarm(
+      this,
+      pagerDuty,
+      this.sqsDlq.name,
+      'prospect-event-rule-dlq-alarm'
```

Comment:
same here with prefix string instead of var

Reply:
added event name